src/platforms/esp32/partitions.csv

```diff
@@ -8,5 +8,5 @@
 nvs,      data, nvs,       0x9000,     0x6000,
 phy_init, data, phy,       0xf000,     0x1000,
 factory,  app,  factory,  0x10000,   0x1C0000,
-lib.avm,  data, phy,     0x1D0000,    0x40000,
+boot.avm,  data, phy,     0x1D0000,    0x40000,
```

Comment:
If we are changing the partition scheme, should we bite the bullet and create (at least) two main.avm partitions?  Otherwise, your new `{atomvm, boot_path}` NVS entry doesn't do anything, unless you manually edit this file.

Reply:
I think that there is no general agreement about enabling the double partitioning schema by default. I think we might provide an example alternative partitions.csv that is suited for OTA.

Reply:
Yeah fair enough.  It means that in order to support “soft” OTA you will need to build your own image.